packages/lesswrong/components/Layout.jsx

```diff
@@ -71,6 +71,7 @@ const Layout = ({currentUser, children, currentRoute, params, client, classes},
             <link name="material-icons" rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/icon?family=Material+Icons"/>
             <link name="react-instantsearch" rel="stylesheet" type="text/css" href="https://unpkg.com/react-instantsearch-theme-algolia@4.0.0/style.min.css"/>
             <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500"/>
+            { theme.typography.fontDownloads && theme.typography.fontDownloads.map((url)=><link rel="stylesheet" href={url}/>) }
```

Comment:
This gives me some hesitation, but might just be fine. I think it would be better to directly expose a hatch to the head element, but I don't know whether that's easily possible.